streamx-plugin/streamx-flink-packer/src/test/scala/com/streamxhub/streamx/flink/packer/MavenToolSpec.scala

```diff
@@ -35,17 +34,15 @@ import scala.language.postfixOps
 class MavenToolSpec extends AnyWordSpec with BeforeAndAfterAll with Matchers {
 
   val outputDir = "MavenToolSpec-output/"
-  val preWorkSpaceVal: String = System.getProperties.getProperty(KEY_STREAMX_WORKSPACE)
+  val preWorkSpaceVal: String = "/streamx"
```

Comment:
Plz restore this value back to `SystemPropertyUtils.get(KEY_STREAMX_WORKSPACE_LOCAL)`, as this is to facilitate direct redirection to the workspace directory during the testing phase without system constraints : )

Reply:
nice.